README.md

```
@@ -48,7 +48,8 @@
 ---
 
 <div align="center">
-<h1>TODO: Project Name</h1>
+<img src="https://raw.githubusercontent.com/AOSSIE-Org/SmartNotes/main/public/logo-icon.svg" width="140" />
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)